CVLByExample/summary/with-env/WithEnvCVLFunctionSummary/README.md

```diff
@@ -0,0 +1,9 @@
+# Function Summarization with env
+
+## test.spec
+Includes a function summary with the flag "with env".
```

Comment:
it's not a flag. maybe optional attribute\feature ?

Reply:
changed